samples/subsys/usb/hid-cdc/prj.conf

```diff
@@ -14,6 +14,7 @@ CONFIG_USB_DEVICE_LOG_LEVEL_INF=y
 
 CONFIG_USB_CDC_ACM=y
 CONFIG_USB_CDC_ACM_DEVICE_COUNT=2
+CONFIG_USB_REQUEST_BUFFER_SIZE=192
```

Comment:
I think its not related to this PR and should be removed or am I missing something?